2a_model/src/models/0_baseline_LSTM/config.yml

```diff
@@ -1,6 +1,8 @@
 exp_name: "0_baseline_LSTM"
 
-x_vars: ['seg_ccov', 'seg_rain', 'seg_slope', 'seg_tave_air', 'seginc_swrad', 'hru_slope', 'hru_aspect', 'hru_elev', 'hru_area', 'hru_percent_imperv', 'covden_sum', 'covden_win', 'soil_moist_max']
+site_set: "well_obs"
+
+x_vars: ['pr','SLOPE','tmmx','srad','CAT_BASIN_SLOPE','CAT_ELEV_MEAN','CAT_BASIN_AREA','CAT_IMPV11', 'CAT_CNPY11_BUFF100','CAT_TWI']
```

Comment:
Ahh! This makes a lot more sense now. We should probably add a comment somewhere in `2a_model.R` (maybe above `p2a_med_obs_data` and `p2a_well_obs_data`) that reminds a user that `x_vars` needs to be updated in this config file if any variables are added/omitted.

Reply:
Good idea! I added a comment in `2a_model` above `p2a_well_obs_data`